### PR TITLE
pulseaudio: add page

### PR DESCRIPTION
**Pull request by sbrl**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by pxgamer** [comment]
Just interested, did you mean to check the `The page has 8 or fewer examples.` checkbox?

File: pages/linux/pulseaudio.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+> The pulseaudio sound system daemon and manager.
+
+- Check if pulseaudio is running (a non-zero exit code means it is not running):
+
+`pulseaudio --check`
+
+- Start the pulseaudio daemon in the background:
+
+`pulseaudio --start`
```
Comment (agnivade) [inline pages/linux/pulseaudio.md]: Should we add "sudo" ?

Comment (sbrl) [reply pages/linux/pulseaudio.md]: Nope! It's a user-mode daemon.

**Comment by sbrl** [comment]
@pxgamer good catch, thanks! Yeah, I did mean to tick it :P

File: pages/linux/pulseaudio.md (hunk 1 above)
Comment (agnivade) [reply pages/linux/pulseaudio.md]: True.

**Pull request merged**